src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ListViewBase.cs

```diff
@@ -655,6 +655,254 @@ public async Task When_Scrolled_To_End_And_Last_Item_Removed()
 			}
 		}
 
+		[TestMethod]
+		[RunsOnUIThread]
+		public async Task When_Items_Their_Own_Container()
+		{
+			var list = new OnItemsChangedListView();
+			var items = new ObservableCollection<ListViewItem>()
+			{
+				new ListViewItem(),
+				new ListViewItem(),
+				new ListViewItem(),
+				new ListViewItem(),
+			};
+
+			list.ItemsSource = items;
+			WindowHelper.WindowContent = list;
+			await WindowHelper.WaitForLoaded(list);
+
+			// Containers/indices/items can be retrieved
+			Assert.AreEqual(items[1], list.ContainerFromItem(items[1]));
+			Assert.AreEqual(items[2], list.ContainerFromIndex(2));
+			Assert.AreEqual(3, list.IndexFromContainer(items[3]));
+			Assert.AreEqual(items[1], list.ItemFromContainer(items[1]));
```

Comment:
When doing a bunch of assertions in a test, you can use FluentAssertion's `AssertionScope()` to execute them all instead of a test failing on the first test.

The syntax would be something like:
```suggestion
			using var _ = new AssertionScope();
			list.ContainerFromItem(items[1]).Should().Be(items[1]);
			list.ContainerFromIndex(2).Should().Be(items[2], );
			list.IndexFromContainer(items[3]).Should().Be(3);
			list.ItemFromContainer(items[1]).Should().Be(items[1]);
```